ajax/libs/jquery.sumoselect/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.sumoselect",
+  "version": "3.0.2",
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
+  "filename": "jquery.sumoselect.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/HemantNegi/jquery.sumoselect.git"
+  },
+  "keywords": [
+    "multiselect",
+    "dropdown"
+  ],
+  "author": "HemantNegi",
+  "license": "MIT",
+  "homepage": "https://hemantnegi.github.io/jquery.sumoselect/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/HemantNegi/jquery.sumoselect.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.sumoselect.!(*json)"
```

Comment:
You should also modify `package.json` to include `sumoselect.css` in other versions.